server/http_handler.go

```diff
@@ -273,6 +275,31 @@ func (t *tikvHandlerTool) handleMvccGetByHex(params map[string]string) (interfac
 	return t.getMvccByEncodedKey(encodedKey)
 }
 
+func (t *tikvHandlerTool) GetAllHistoryDDL() ([]*model.Job, error) {
```

Comment:
I think we do not need to expose this method.

Reply:
If we do not need to expose this method, then getting ddl job history is trouble.